clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -82,13 +82,22 @@
   </xs:complexType>
 
   <xs:complexType name="connection-spec">
-    <xs:attribute name="url" type="tc:connectionUrl" use="required">
+    <xs:attribute name="url" type="tc:connectionUrl">
       <xs:annotation>
         <xs:documentation xml:lang="en">
           Identifies a single cluster member by URL.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="servers" type="tc:servers">
+      <xs:annotation>
+        <xs:documentation xml:lang="en">
+          Identifies servers in the cluster.
+        </xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+    <xs:attribute name="cacheManager" type="xs:string">
```

Comment:
If we have different element types for the two configuration types then we can make this attribute required on the server list form.